config/ldp/metadata-writer/writers/modified.json

```
@@ -0,0 +1,10 @@
+{
+  "@context": "https://linkedsoftwaredependencies.org/bundles/npm/@solid/community-server/^1.0.0/components/context.jsonld",
+  "@graph": [
+    {
+      "comment": "Adds the correct Last-Modified and ETag headers.",
```

```suggestion
      "comment": "Adds the Last-Modified and ETag headers.",
```

wouldn't want to add fake ones now, would we? 😛